docs/setup/native_setup.md

```diff
@@ -28,13 +80,13 @@ git remote add upstream https://github.com/OperationCode/operationcode_backend.g
 
 #### Database Setup:
 
-
+1a. Run redis in separate tab.
 1. Run `cp config/database.yml. config/native_setup_database.yml`.
```

Comment:
The current database.yml allows for passing a different host in via `POSTGRES_HOST`. I would recommend not having contributors replace the `database.yml` since it will likely be committed in PRs.

Reply:
What would you recommend instead?

Reply:
Passing the posgres host via an ENV variable. `POSTGRES_HOST=localhost bundle exec rails db:migrate`